internal/cli/organizations/invitations/update.go

```diff
@@ -132,7 +133,8 @@ func UpdateBuilder() *cobra.Command {
 	opts.AddOrgOptFlags(cmd)
 	opts.AddOutputOptFlags(cmd)
 
-	_ = cmd.MarkFlagRequired(flag.Role)
+	_ = cmd.MarkFlagFilename(flag.File)
+	cmd.MarkFlagsMutuallyExclusive(flag.File, flag.Role)
```

Comment:
[q] this is great, but does it become a breaking change?

Reply:
As far as I know, this won't become a breaking change.

The reason why I changed this is that if you provided a role using `--file` you were still required to set `--role`.